src/styles/form.scss

```diff
@@ -0,0 +1,50 @@
+.form {
+  margin-bottom: 48px;
+
+  @include onTablet {
+    margin-bottom: 80px;
+  }
+
+  @include onDesktop {
+    margin-bottom: 0;
+  }
+
+  &-field {
+    width: 100%;
+    box-sizing: border-box;
+    padding: 14px 16px;
+    border-radius: 8px;
+    background: #f8f8fa;
+    border: none;
+
+    &::placeholder {
+      @include form-plaseholder-style;
+    }
+
+    &:hover {
+      border: 1px solid #d5d7de;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
9. [STYLES] - Don't add new `border` to the element on hover. Add default transparent border of the same width, and change its color on `:hover`